extensions/examples/online-extensions.js

```diff
@@ -41,8 +41,8 @@ angular
     'extensionRegistry',
     function(extensionRegistry) {
 
-      var system_status_elem = $('<a href="http://status.openshift.com/" target="_blank" class="nav-item-iconic system-status project-action-btn">');
-      var system_status_elem_mobile = $('<div row flex class="navbar-flex-btn system-status-mobile">');
+      var system_status_elem = $('<a href="http://status.openshift.com/" target="_blank" class="nav-item-iconic system-status project-action-btn" style="display: none;">');
+      var system_status_elem_mobile = $('<div row flex class="navbar-flex-btn system-status-mobile" style="display: none;">');
```

Comment:
I'm unclear why we set `display: none` and then reset it below. It looks like the DOM elements are only appended when there's an issue, so it should be unnecessary?